app/models/department.rb

```diff
@@ -0,0 +1,12 @@
+class Department < ApplicationRecord
+
+  scope :sorted, lambda {order("position ASC")}
+
+  CAPACITY = *(50..550)
+
+  validates :name, :presence => true, :uniqueness => true
+  validates :position, :presence => true, :length => {minimum: 1}, :uniqueness => true, :numericality => true
```

Comment:
Always follow the new Hash syntax `presence: true`, not `:presence => true`.
Same instruction for rest of the hashes